conf/config_template.yaml

```diff
@@ -116,9 +115,7 @@ inference:
   chunk_size: 512 # (int) Size (height and width) of each prediction patch. Default: 512
   overlap: 10 # (int) Percentage of overlap between 2 chunks. Default: 10
   heatmaps: True
-
-
-# Visualization parameters
+  # Visualization parameters
   smooth_prediction: True
   overlap: 2 # overlap between tiles for smoothing. Must be an even number that divides chunk_size without remainder.
```

Comment:
overlap is used twice as a parameter in the inference section. I'm getting yaml warnings from this. Probably a bug.

Reply:
Yeah its a new issues that we need to talk.